docs/README.md

```diff
@@ -349,6 +349,13 @@ You will first need to run the following command (from the root of the repositor
 python utils/prepare_for_doc_test.py src docs
 ```
 
+If you work on a specific python script, say `modeling_wav2vec2.py`, you can run the command as follows (to avoid the unnecessary temporary changes in irrelevant files):
```

Comment:
```suggestion
If you work on a specific python module, say `modeling_wav2vec2.py`, you can run the command as follows (to avoid the unnecessary temporary changes in irrelevant files):
```
`script` implies that it should be run, which is something that is not encouraged/will generally not work.